src/main/java/net/atos/zac/app/taken/TakenRESTService.java

```diff
@@ -260,18 +262,12 @@ public RESTTaak toekennenAanIngelogdeMedewerker(final RESTTaakToekennenGegevens
         return restTaakConverter.convert(task);
     }
 
-    @PATCH
-    public RESTTaak updateTaak(final RESTTaak restTaak) {
+    private Task updateTaak(RESTTaak restTaak) {
```

Comment:
This endpoint was never called from the front end, so I made it a private method. I felt asserting policies and sending out events should be the responsibility of endpoints, so removed that code from this method.

Reply:
It was being used by updateTaakdata, which is why I made it private in stead of removing it. But the suggestion you make in your other comment might make it possible to remove it after all.